airbyte-config/models/src/main/resources/types/WorkspaceServiceAccount.yaml

```diff
@@ -0,0 +1,27 @@
+---
+"$schema": http://json-schema.org/draft-07/schema#
+"$id": https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/WorkspaceServiceAccount.yaml
+title: WorkspaceServiceAccount
+description: service account attached to a workspace
+type: object
+required:
+  - workspaceId
+  - serviceAccountId
+  - serviceAccountEmail
+additionalProperties: false
+properties:
+  workspaceId:
+    type: string
+    format: uuid
+  serviceAccountId:
+    type: string
+  serviceAccountEmail:
+    type: string
+  jsonCredential:
```

Comment:
Nitpick. It would be helpful to add a sample json credential here, or just add a reference to the `mock_serivce_key.json` file. Sometimes it is hard to find how a json column should look like.

Reply:
could this just be a string type? I.e. do we ever actually want to deserialize the cred; my understanding is that the fact that it's a JSON blob is just an implementation detail of GCP and we shouldn't need to care about it

regardless, I do agree that an example would be useful for future reference

Reply:
^ agree with this. The pro for having a json column is that it is easier to search. However, I imagine that in reality we seldom have to search anything from the credentials. So there is not much benefit from storing the json type here.

Reply:
I have added a comment for reference to the example, I think there is more value in keeping this as JsonNode data type cause it makes sure that the data type is in sync with the database + the key of service account is in JSON format